package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "esphome-webserver",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
I don't thing you should change the version number. This will be done when creating a new release.